docs/sdks/common-issues.md

```diff
@@ -2,7 +2,7 @@
 sidebar_position: 9
 ---
 
-# Best practices
+# Common pitfalls
```

Comment:
Renaming this page because the sections are things _not_ to do, not best practices.

Reply:
Good catch 📈